packages/strapi-plugin-email/admin/src/translations/en.json

```diff
@@ -5,7 +5,7 @@
   "EditForm.Input.select.inputDescription": "Emails can be sent with the default provider (Sendmail) or an external provider",
   "EditForm.Input.select.label": "Providers",
   "EditForm.Input.toggle.label": "Enable email send",
-  "notification.config.success": "The settings has been updated",
+  "notification.config.success": "The settings have been updated",
```

Comment:
You have a conflict on this file can you remove all the translation except the two last one?

Reply:
Thank you, I've updated this branch to the latest master.